crossmatch/ztf_ps1_crossmatch.md

```diff
@@ -0,0 +1,192 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: notebook
+  language: python
+  name: python3
+---
+
+# Cross-Matching ZTF and Pan-STARRS using LSDB 
+
++++
+
+# Learning Goals
+
+By the end of this tutorial, you will:
+- understand how to cross-match cloud-based catalogs using `lsdb`.
+- understand how to parallelize `lsdb` cross-matches using `dask`.
+- have a feeling for when `dask` parallelization can be helpful.
+
++++
+
+# Introduction
+
+[LSDB](https://lsdb.io) is a useful package for performing large cross-matches between source catalogs. It's built to run across multiple nodes with Dask parallelization, but even without parallelization it is high-performance. Here we will benchmark the performance of LSDB on the NASA Fornax platform with and without Dask.
+
+We will start small, trying to cross-match 10,000 sources from ZTF with Pan-STARRS. We will then scale up by factors of roughly 10 until either (a) the platform can no longer handle the load, or (b) we do the full cross-match.
+
+For each level, we want to know the performance with (1) no Dask, (2) minimal Dask - like 2 workers, (3) bigger Dask - as many workers as we can use, and (4) auto-scaling Dask.
+
++++
+
```

Comment:
Please add a "Runtime" section to the intro.  You can use this notebook for an example of what that should look like: https://github.com/nasa-fornax/fornax-demo-notebooks/blob/main/light_curves/light_curve_generator.md

or the tech review checklist under "documentation" has an example.